virttest/ip_sniffing.py

```diff
@@ -167,7 +167,10 @@ def _output_logger_handler(self, line):
         # We can check whether the process is terminated unexpectedly
         # here since the terminated status is a line of the output
         if re.match("\(Process terminated with status", line):
-            status = self._process.get_status()
+            if wait_for(lambda: self._process, 1, 0.1):
+                status = self._process.get_status()
+            else:
+                status = "<not available, self._process not set>"
```

Comment:
How about capturing the `status` from log message, like:
```python
match = re.match("\(Process terminated with status (\d+)", line):
if match:
    status = match.group(1)
```
thus it won't reference `self._process` any more.

Reply:
yep, makes sense and the probability of false-match is really low (that's what would be indicated by "not available..."). I'll change it (probably tomorrow, right now I have some other business)